docs/operator-manual/upgrading/2.3-2.4.md

```diff
@@ -24,18 +24,17 @@ The signature algorithm is negotiated with the SSH server when the connection is
 list of accepted signature algorithms, and if the server has a match, the connection proceeds. For most SSH servers on
 up-to-date git providers, acceptable algorithms other than `ssh-rsa` should be available.
 
-Before upgrading to Argo CD 2.4, check whether your git provider(s) using SSH authentication support algorithms newer
-than `rsa-ssh`.
+As long as `ssh_rsa` is not supported as single signature algorithm in Gitlab server, OpenSSH_8.9 is not mandatory on server side. Fallback to alternative OpenSSH version is acknowledged like this:
 
-1. Make sure your version of SSH >= 8.9 (the version used by Argo CD). If not, upgrade it before proceeding.
+```debug1: Local version string SSH-2.0-OpenSSH_8.9p1 Ubuntu-3
```

Comment:
Does the text of this line need to be after a newline? Not 100% certain, but I think this might be interpreted as a language annotation.

Reply:
Agreed

Reply:
One new line removal, the first line _debug1: Local version string SSH-2.0-OpenSSH_8.9p1 Ubuntu-3_ is removed also